app/views/my_facilities/drug_stocks/new.html.erb

```diff
@@ -31,10 +31,13 @@
             <%= drug_stock_form.hidden_field :protocol_drug_id, value: protocol_drug.id %>
             <div class="form-row">
               <div class="col">
-                <%= drug_stock_form.number_field :received, class: :received, value: @drug_stocks[protocol_drug.id].try(&:received), hide_label: true, help: "Received during the month" %>
```

Comment:
Were we using the `class: :received` for any styling?

Reply:
No, this was unused.